src/avro/src/decode.rs

```diff
@@ -7,22 +7,22 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use std::fmt::Display;
 use std::mem::transmute;
+use std::{
+    collections::HashMap,
+    io::{Read, Seek, SeekFrom},
+};
```

Comment:
is this intentional? I thought we used the [module style](https://github.com/rust-lang/rustfmt/issues/3362#issuecomment-642564290) of import in general?

FWIW I prefer module imports because it makes things like `rg std.*Read` guaranteed to succeed, but I don't object if folks prefer other things.

Reply:
We do! I was just thoughtlessly preserving what was already there! Fixed, thanks.